### ISSUE TITLE
By what properties do you differentiate an experienced developer (React)?

### ISSUE THREAD

**Issue by adtm** [kentcdodds/ama#456]
This is more about front-end related, f.e: React (the most favorite thing in the world 😱).

This is not about patterns you can use, but more about coding style and architecture decision. F.e: maybe more advanced programmers don't make plain `axios` calls in their `componentDidMount`, but make a separate API abstraction which takes care of it, maybe someone uses static analysis tools like Prettier or ESLint enhancing their workflow.

So, in your opinion, what decision an experienced React developer will take, which may not come at first sight?

**Comment by kentcdodds** [comment]
Hi! I'm afraid I don't have time to give a super solid answer to all my open questions right now, so I just went through and answered all my AMA questions in the form of a video: https://www.youtube.com/watch?v=4Yq6PO0r-fU&list=PLV5CVI1eNcJgCrPH_e6d57KRUTiDZgs0u

A few things that folks can do to help make things better for others reading the questions and answers:

1. Link to when I start answering the question. Add `&t=xxmxxs` to the end of the URL like this: https://www.youtube.com/watch?v=4Yq6PO0r-fU&list=PLV5CVI1eNcJgCrPH_e6d57KRUTiDZgs0u&t=2m5s
2. Comment with links I referenced
3. Summarize what I talk about

Thanks!
